tests/xtimer_usleep_short/tests/01-run.py

```diff
@@ -12,19 +12,27 @@
 
 
 def testfunc(child):
-    child.expect(u"This test will call xtimer_usleep for values from \\d+ down to \\d+\r\n")
+    child.expect(u"This test will call xtimer_usleep for values from (\d+) down to (\d+)")
+    sleep_max = int(child.match.group(1))
+    sleep_min = int(child.match.group(2))
 
-    i = 500
+    i = sleep_max - sleep_min
 
     while (i >= 0):
         try:
-            child.expect(u"going to sleep \\d+ usecs...\r\n", timeout=3)
+            child.expect(u"going to sleep (\d+) us", timeout=3)
```

Comment:
Here you can replace `(\d+)` by the value of `i` or if renamed `delay`. It would prevent issue when testing with text taken from node before `make reset`.